frontend/src/components/Psychologist/Profile/PsyProfile.jsx

```diff
@@ -26,17 +26,41 @@ const PsyProfile = () => {
   const viewProfilRef = useRef();
   const suspensionInfoRef = useRef();
 
-  const { commonStore: { setNotification }, userStore: { pullUser } } = useStore();
+  const { commonStore: { setNotification }, userStore: { pullUser, user } } = useStore();
   const history = useHistory();
   const [loading, setLoading] = useState(true);
   const [psychologist, setPsychologist] = useState();
   const [editMode, setEditMode] = useState(false);
   const [suspensionMode, setSuspensionMode] = useState(false);
 
+  const getProfilIssues = psy => {
+    const profilIssues = [];
+    if (!psy.description || psy.description.length < 50) {
+      profilIssues.push('Votre présentation est trop courte.');
+    }
+
+    if (!psy.longitude || !psy.latitude) {
+      profilIssues.push('Votre adresse ne semble pas valide.');
+    }
+
+    const isWebsite = new RegExp('^(https?:\\/\\/)?' // protocol
+    + '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,}|' // domain name
+    + '((\\d{1,3}\\.){3}\\d{1,3}))' // OR ip (v4) address
+    + '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*' // port and path
+    + '(\\?[;&a-z\\d%_.~+=-]*)?' // query string
+    + '(\\#[-a-z\\d_]*)?$', 'i'); // fragment locator
+    if (!isWebsite.test(psy.website)) {
```

Comment:
If we check website url, do we still need to add `http://` before a string without protocol? (see `string.prefixUrl`)

Reply:
We are not checking anymore... it's just an info :/